torch/csrc/jit/source_range.h

```diff
@@ -117,6 +117,20 @@ struct CAFFE2_API SourceRange {
     return ss.str();
   }
 
+  c10::optional<std::tuple<std::string, size_t, size_t>> file_line_col() const {
```

Comment:
perf: This is creating a copy of the filename string for _every_ line of the output.